src/components/Header.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './styles/header.css';
+
+function Header() {
+  return (
+    <nav>
+      <div className="navbar links flex">
+        <ul className="flex">
+          <li>
+            <h2>
+              <a href="/books" className="logo">Bookstore CMS</a>
+
+            </h2>
+          </li>
+          <li>
+            <NavLink to="/books">
+              Books
+            </NavLink>
+          </li>
```

Comment:
- [ ] Kindly ensure the Navbar link matches the root path "/" in the App component.